pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.styles.ts

```diff
@@ -1,32 +1,37 @@
+import { GrafanaTheme } from '@grafana/data';
 import { css } from 'emotion';
 
-export const availableUpdate = css`
-  align-items: flex-start;
-  display: flex;
-  font-weight: bold;
-  justify-content: flex-start;
-  line-height: 1.2;
-  margin-top: 5px;
-
-  > div {
+export const getStyles = ({ spacing, typography }: GrafanaTheme) => ({
+  availableUpdate: css`
+    align-items: flex-start;
     display: flex;
-  }
-
-  a {
-    margin-left: 5px;
-  }
-`;
-
-export const availableUpdateVersion = css`
-  font-size: 14px;
-  margin-bottom: 7px;
-`;
+    font-weight: bold;
```

Comment:
should we use `typography.weight.bold` ?

Reply:
Done!